asserts/database_test.go

```diff
@@ -774,6 +774,72 @@ func (safs *signAddFindSuite) TestFindTrusted(c *C) {
 	c.Check(err, Equals, asserts.ErrNotFound)
 }
 
+func (safs *signAddFindSuite) TestFindManyTrusted(c *C) {
+	trustedKey0 := testPrivKey0
+	trustedKey1 := testPrivKey1
+	cfg := &asserts.DatabaseConfig{
+		Backstore: asserts.NewMemoryBackstore(),
+		Trusted: []asserts.Assertion{
+			asserts.BootstrapAccountForTest("canonical"),
+			asserts.BootstrapAccountKeyForTest("canonical", trustedKey0.PublicKey()),
+			asserts.BootstrapAccountKeyForTest("canonical", trustedKey1.PublicKey()),
+		},
+	}
+	db, err := asserts.OpenDatabase(cfg)
+	c.Assert(err, IsNil)
+
+	pk1 := testPrivKey2
+
+	acct1 := assertstest.NewAccount(safs.signingDB, "acc-id1", map[string]interface{}{
+		"authority-id": "canonical",
+	}, safs.signingKeyID)
+
+	acct1Key := assertstest.NewAccountKey(safs.signingDB, acct1, map[string]interface{}{
+		"authority-id": "canonical",
+	}, pk1.PublicKey(), safs.signingKeyID)
+
+	err = db.Add(acct1)
+	c.Assert(err, IsNil)
+	err = db.Add(acct1Key)
+	c.Assert(err, IsNil)
+
+	// find the trusted account
+	tAccts, err := db.FindManyTrusted(asserts.AccountType, map[string]string{
+		"account-id": "canonical",
+	})
+	c.Assert(err, IsNil)
+	c.Assert(tAccts, HasLen, 1)
+	c.Assert(tAccts[0].(*asserts.Account).AccountID(), Equals, "canonical")
```

Comment:
Could just use HeaderString("account-id") to avoid the type assertion if you prefer? I tend to find that more readable in tests. Ditto below for account keys.